src/main/kotlin/io/snyk/plugin/net/SnykApiClient.kt

```diff
@@ -10,8 +10,8 @@ import retrofit2.Retrofit
 class SnykApiClient constructor(
     private val retrofit: Retrofit
 ) {
-    val sastOnServerEnabled: Boolean?
-        get() = executeRequest(CliConfigService.apiName, cliConfigService().sast())?.sastEnabled
+    val sastSettings: CliConfigSettings?
+        get() = executeRequest(CliConfigService.apiName, cliConfigService().sast())
```

Comment:
Another thing that is missing (but we already have a ticket for this), is the transmission of the organisation as a query parameter.

Reply:
agreed, let's do it in another ticket.